lms/static/js/student_account/views/account_settings_factory.js

```diff
@@ -83,8 +84,14 @@
                                 model: userAccountModel,
                                 title: gettext('Full Name'),
                                 valueAttribute: 'name',
-                                helpMessage: gettext(
-                                    'The name that is used for ID verification and appears on your certificates. Other learners never see your full name. Make sure to enter your name exactly as it appears on your government-issued photo ID, including any non-Roman characters.'  // eslint-disable-line max-len
+                                helpMessage: HtmlUtils.interpolateHtml(
+                                    gettext(
+                                        'The name that is used for ID verification and appears on your certificates. Other learners see your full name if you have selected {bold_start}Full Profile{bold_end} for profile visibility. Make sure to enter your name exactly as it appears on your government-issued photo ID, including any non-Roman characters.'  // eslint-disable-line max-len
```

Comment:
@robrap  can we add "that" before "appears", and delete "government-issued" in front of photo ID (we no longer require that). 
"The name that is used for ID verification and that appears on your certificates. ... Make sure to enter your name exactly as it appears on your photo ID..."

Reply:
Done.  I also squashed.